SwissTransferTests/SwissTransferTests.swift

```diff
@@ -17,6 +17,46 @@
  */
 
 import Foundation
+@testable import STNewTransferView
 import XCTest
 
-final class SwissTransferTests: XCTestCase {}
+@MainActor
+final class SwissTransferTests: XCTestCase {
```

Comment:
I do not see a teardown or a setup method. I believe you said you wanted to clean before and after each test.